var/spack/repos/builtin/packages/opentsdb/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Opentsdb(AutotoolsPackage):
+    """
+    OpenTSDB is a distributed, scalable Time Series Database (TSDB) written on
+    top of HBase.  OpenTSDB was written to address a common need: store, index
+    and serve metrics collected from computer systems (network gear, operating
+    systems, applications) at a large scale, and make this data easily
+    accessible and graphable.
+    """
+
+    homepage = "https://github.com/OpenTSDB"
+    url      = "https://github.com/OpenTSDB/opentsdb/archive/v2.4.0.tar.gz"
+
+    def url_for_version(self, version):
+        url = "https://github.com/OpenTSDB/opentsdb/archive/v{0}.tar.gz"
+        return url.format(version)
```

Comment:
You don’t actually need this, Spack should be able to handle this url automatically.